.bumpversion.cfg

```diff
@@ -1,7 +1,6 @@
 [bumpversion]
-current_version = 2.0.0
+current_version = 2.0.1
```

Comment:
This is a backwards incompatible change so this should be a major bump

Reply:
Bumped major version